dev/tests/integration/testsuite/Magento/UrlRewrite/Model/StoreSwitcher/RewriteUrlTest.php

```diff
@@ -65,8 +65,8 @@ protected function setUp(): void
     /**
      * Test switching stores with non-existent cms pages and then redirecting to the homepage
      *
-     * @magentoDataFixture Magento/UrlRewrite/_files/url_rewrite.php
      * @magentoDataFixture Magento/Catalog/_files/category_product.php
+     * @magentoDataFixture Magento/UrlRewrite/_files/url_rewrite.php
```

Comment:
Looks like this file could be reverted.

Reply:
In this case, changing the order of fixture loading have sense, as product save generates rewrites for all assigned websites. Then can not be checked logic of redirect from unexisting rewrite to home page.